.github/workflows/flash_analysis.yml

```diff
@@ -52,30 +59,39 @@ jobs:
         run: make submodulesupdate
 
       - name: Build
-        run: make ${{ matrix.target }}
+        run: make ${{ matrix.target }}_flash-analysis
```

Comment:
It looks like it's failing to find this target in the latest CI run. My guess is you need to check-out the right commit before this step. Can you please verify? @alexcekay

Reply:
Hi. This is due to the following:
> FLASH analysis on this PR will fail (as the base branch does not yet have the flash_analysis target), but once merged it will work on other branches (as the main will then have that target).

Basically when checking out the base of this PR, the target _flash-analysis_ did not exist yet as this target will be added by this PR. After merging the change this won't be a problem. If we want that it also works in this PR, we need to close this PR, create another PR that adds the _flash-analysis_ target, merge the PR that adds the _flash-analysis_ target and create again this PR afterwards.